src/analysis/retail/monk/mistweaver/modules/core/HotAttributor.ts

```diff
@@ -95,16 +102,18 @@ class HotAttributor extends Analyzer {
           'on ' + this.combatants.getEntity(event)?.name,
           ' expected expiration: ' +
             this.owner.formatTimestamp(
-              event.timestamp + Number(this.hotTracker.hotInfo[event.ability.guid].procDuration),
+              event.timestamp +
+                Number(this.hotTracker.hotInfo[SPELLS.RENEWING_MIST_HEAL.id].procDuration),
               3,
             ),
         );
       this.hotTracker.addAttributionFromApply(this.rapidDiffusionAttrib, event);
-      this.hotTracker.hots[event.targetID][event.ability.guid].maxDuration = Number(
-        this.hotTracker.hotInfo[event.ability.guid].procDuration,
+      this.hotTracker.hots[event.targetID][SPELLS.RENEWING_MIST_HEAL.id].maxDuration = Number(
```

Comment:
Redundant, the event listener is only looking for rem id

Reply:
Ya they're equivalent, I just think the code looks cleaner with hardcoding REM id lol. I can change it back if you want tho